lib/src/blocs/coins_bloc/coins_event.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:equatable/equatable.dart';
+
+abstract class CoinsEvent extends Equatable {
+  @override
+  List<Object?> get props => [];
+}
+
+class GetListCoins extends CoinsEvent {
+  String? currency;
+  bool? sparkline;
+
+
+  GetListCoins({ this.currency, this.sparkline});
+}
```

Comment:
we should override maybe like this
List<Object?> get props => [currency,sparkline];

Reply:
fixed